libia2/include/pkey_init.h

```diff
@@ -0,0 +1,30 @@
+#ifndef PKEY_INIT_H
```

Comment:
Previously we generated this file to call the rust libia2, but we don't need this file (ia2.h already has the things it defines/declares).